pages/_app.js

```diff
@@ -23,7 +23,7 @@ function MyApp({Component, pageProps}) {
       <Component
         {...pageProps}
         loginName={loginName}
-        onSubmitLogin={onSubmitLogin}
+        handleSubmitLogin={handleSubmitLogin}
         userArray={userArray}
```

Comment:
Here you renamed the prop as well - the prop name was correct though: `onSubmitLogin`
So the line should read: `onSubmitLogin={handleSubmitLogin}`